docs/src/main/paradox/journal.md

```diff
@@ -8,6 +8,29 @@
 - Messages written by a single persistent actor are partitioned across the cluster to achieve scalability with data volume by adding nodes.
 - @extref:[Persistence Query](akka:scala/persistence-query.html) support by `CassandraReadJournal`
 
+### Schema 
+
+The keyspace and tables needs to be created before using the plugin. Auto creation of the keyspace and tables
+is included as a development convenience and should never be used in production. Cassandra does not handle
+concurrent schema migrations well and if every Akka node tries to create the schema at the same time you'll
+get column id mismatch errors in Cassandra.
```

Comment:
Perhaps even put the caution in a warning callout to really make sure users notice?